.github/workflows/automation.yml

```diff
@@ -0,0 +1,67 @@
+name: Automation
+
+on:
+  pull_request:
+  issues:
+  issue_comment:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Does PR has the stable backport label?
+        uses: Dreamcodeio/does-pr-has-label@v1.2
+        id: checkLabel
+        with:
+          label: stable backport
+
+      - name: Remove from 'Current Release' project
+        uses: alex-page/github-project-automation-plus@v0.5.1
+        if: (github.event.pull_request || github.event.issue.pull_request) && !steps.checkLabel.outputs.hasLabel
+        continue-on-error: true
+        with:
+          project: Current Release
+          column: In progress
```

Comment:
```suggestion
```

Looking at the source the `column` isn't passed in
https://github.com/alex-page/github-project-automation-plus/blob/main/src/generate-mutation-query.js#L73-L80